articles/install-guide/csinstall.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Develop with Q# + C#
+author: natke
+ms.author: nakersha
+ms.date: 9/30/2019
+ms.topic: article
+ms.custom: how-to
+uid: microsoft.quantum.install.cs
+---
+# Develop with Q# + C#
+
+Install the QDK to develop C# host programs to call Q# operations.
+
+Q# is a member of the .NET ecosystem and therefore is built to play well with other .NET languages--specifically C#. You can work with this pairing inside different development environments:
+
+- Q# + C# using Visual Studio (Windows)
+- Q# + C# using Visual Studio Code (Windows, Linux and Mac)
+- Q# + C# using the `dotnet` command-line tool
+
+## Develop with Q# + C# using Visual Studio
+
+Visual Studio offers a rich environment for developing Q# programs. The Q# Visual Studio extension contains templates for Q# files and projects as well as syntax highlighting, code completion and IntelliSense support.
+
+
+1. Pre-requisites
+
+    - [Visual Studio](https://visualstudio.microsoft.com/downloads/) 16.3, with the .NET Core cross-platform development workload enabled
```

Comment:
@cgranade, @bettinaheim @anpaz-msft how do we feel about calling out version number here? Are we limited by this, or can we say to make sure to have the latest version of Visual Studio?

Reply:
My own 2¢ is that it's worth being explicit. VS versions can be finnicky enough that it's good to make sure we've tested with new versions before incrementing here rather than saying "latest."